crates/libcontainer/src/rootfs/utils.rs

```diff
@@ -4,6 +4,18 @@ use oci_spec::runtime::{LinuxDevice, LinuxDeviceBuilder, LinuxDeviceType, Mount}
 use procfs::process::MountInfo;
 use std::path::{Path, PathBuf};
 
+use crate::syscall::linux;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MountOptionConfig {
+    // Mount Flags.
+    pub flags: MsFlags,
+    // Mount data applied to the mount.
+    pub data: String,
+    // RecAttr represents mount properties to be applied recrusively.
+    pub rec_attr: Option<linux::MountAttr>,
```

Comment:
For following cargo docs style
```suggestion
    /// Mount Flags.
    pub flags: MsFlags,
    
    /// Mount data applied to the mount.
    pub data: String,
    
    /// RecAttr represents mount properties to be applied recrusively.
    pub rec_attr: Option<linux::MountAttr>,
```